.github/workflows/cmake.yml

```diff
@@ -47,8 +47,9 @@ jobs:
     - name: Compile
       run: |
         cmake --preset linux-ci-build-with-nonstandard-options -L
-        cmake --build --preset linux-ci-build-with-nonstandard-options --verbose
-        sudo cmake --build --preset linux-ci-build-with-nonstandard-options --target install
+        cd build/linux-ci-build-with-nonstandard-options
+        cmake --build . --verbose
+        cmake --build . --target install
```

Comment:
The `sudo` was there because `/usr/local/lib` is not writable by the CI user on this VM. You'll need to add it back to get the CI passing.

Reply:
Thanks - that's done.